scripts/extractFutureEdition.py

```diff
@@ -0,0 +1,23 @@
+#Extracting the edition number from the has_version field of records for any future records
+def extract(version : str, language : str):
```

Comment:
I think for offhand scripts such as this one, it's a good idea since these are easier to forget over time unlike the other pieces of code we will interact with more often